### PR TITLE
jool: init at 3.3.2

### PR DESCRIPTION
**Pull request by fpletz**
Jool is a SIIT and a NAT64 implementation for Linux.

### CODE REVIEW

**Comment by jagajaga** [comment]
Tested?

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -9507,6 +9507,8 @@ let
 
   jfbview = callPackage ../os-specific/linux/jfbview { };
 
+  jool = callPackage ../os-specific/linux/jool/cli.nix { };
```
Comment (lethalman) [inline pkgs/top-level/all-packages.nix]: Call it `jook-cli` otherwise it's confusing w.r.t. the linux package.

**Comment by fpletz** [comment]
@jagajaga Yup, tested and works.

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (fpletz) [inline pkgs/top-level/all-packages.nix]: Thanks, changed and rebased to current master.

**Pull request merged**